db/model/generatortemplate.js

```diff
@@ -188,6 +189,37 @@ module.exports = function user(seq, dataTypes) {
         return 'generatorTemplateAccess';
       },
 
+      getGTMatchingNameVersion(name, version) {
+        return GeneratorTemplate.findAll({
+          where: {
+            name,
+          },
+        }).then((templates) => {
+          if (!templates || !templates.length) {
+            return null;
+          }
+
+          let matchedTemplate = null;
+          templates.forEach((template) => {
+            if (matchedTemplate) {
+              /*
+              * ok is true when the current template version satisfies the
+              * given version and the current template version is >= greater
+              * then or equal to the version of the matched template
```

Comment:
no need to say ">=" AND "greater then or equal to"

(also, it would be "greater than" rather than "greater then")